docs/GOVERNANCE.md

```diff
@@ -159,53 +165,85 @@ The specifications produced will adhere to the following:
   that the "minor" version number will always be zero and the `specversion`
   string will always be of the form `X.0`.
 
-Note that these rules do not apply to the
-[documented extensions](../cloudevents/extensions/README.md).
+- Each release will have both a tag and a branch. The tag will be kept
+  up-to-date with the tip of the branch. The purpose of having a branch is to
+  support very minor fixes (typos, clarifications) which amend a release in
+  place. The purpose of having a tag is to support GitHub releases, which can
+  act as a notification channel for interested users.
 
-All versions are tagged from the `main` branch, but the tag only applies to
-the "subject" of the release - the directory containing the information
-covered by that release (e.g. `subscriptions` or `cloudevents`). The
-[CloudEvents web site](https://cloudevents.io/) takes appropriate content from
-each tagged version. (If the directory containing the information covered
-by the release is not in a top-level directory, the subject should be the full path,
-e.g. `top-dir/sub-dir`.)
-
-> Note: should the need arise, additional branches may be created. For example,
-> it is likely that a `core-v2.0` branch will be created to collect changes for
-> the core specification version 2.0 significantly before those changes are
-> merged into the main branch, to allow for ongoing work on the main branch.
-> Such branches should be deleted once their content is eventually merged.
+Note that these rules do not apply to unversioned documents, such as the
+[documented extensions](../cloudevents/extensions/README.md).
 
 To create a new release:
 
 - Periodically the group will examine the list of extensions to determine
   if any action should be taken (e.g. removed due to it being stale). The
   creation of a new release will be the reminder to do this check. If any
   changes are needed then PRs will be created and reviewed by the group.
-- Create a PR that modifies the [README](README.md), and all specifications (ie.
-  \*.md files) that include a version string, to the new release version string.
-  Make sure to remove `-wip` from all of the version strings.
+
+- Determine the new release version string. It should be of the form:
+  `<subject>/vX.Y.Z`, e.g. `cloudevents/v1.0.4` or `subscriptions/v1.0.0`.
+
+- Create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string appended with
+    `-rc#` as appropriate. Make sure to remove all `-wip` suffixes as needed.
+  - Merge the PR.
+  - Initiate a final review/test of the release.
+
+- Create A "release candidate" with the new release version string but with a
```

Comment:
It's not clear what "create a release candidate" actually means. What's the concrete step here? Should the "Create a PR" step above actually be a sub-bullet of this one?

Reply:
I'm reworking this section a bit and I think it'll address your concerns